src/views/utility/source-link.js

```diff
@@ -44,87 +48,112 @@ export class FormattedFileName extends LitElement {
           padding-bottom: 6px;
           padding-left: 4px;
         }
+
+        .cbclink {
+          background-color: #333333;
```

Comment:
this is all very nice code and works great as far as I tested it, the only thing I am not sure about is the background color for the CBC link. Maybe we should just go for the same color as the button? the dark background right now is a bit disturbing (at least I think so).

Reply:
I made it the same color as they are using. But I can take all that css code out. We do not need to do the same thing they are doing as they don't have an official logo.